src/runtime/components/Layer.vue

```diff
@@ -79,8 +79,43 @@ useMapbox(mapId, (map) => {
     }
 
     addLayer();
+
+    if ((props.layer as Layer).minzoom) {
```

Comment:
I think this if statement will cause the watcher to only work if the layer has the prop defined on setup (i.e. if you add "minzoom" later but don't have it initially it won't update.) 

Not a big deal though IMO I can always come back and fix later.